Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -1202,6 +1207,40 @@ final class PledgeViewModelTests: TestCase {
         ["Checkout Payment Page Viewed"],
         self.trackingClient.events
       )
+
+      XCTAssertEqual(optimizelyClient.trackedUserId, "DEADBEEF-DEAD-BEEF-DEAD-DEADBEEFBEEF")
```

Comment:
I wonder if it might be nicer to create a separate test for the optimizely events? That way when we remove the experiment we can just delete the test 🤔

Reply:
Can do!